lib/omniauth/login_dot_gov/authorization.rb

```diff
@@ -17,7 +18,7 @@ def redirect_url
       private
 
       def redirect_url_params
-        {
+        params = {
```

Comment:
**suggestion:** don't think you need this assignment any more since it will be bundled in `acr_values`

Reply:
do we not need to include the `aal` parameter?

Reply:
not as far as I'm aware